geoopt/tensor.py

```diff
@@ -100,6 +100,13 @@ def __repr__(self):
             self.manifold
         ) + torch.Tensor.__repr__(self)
 
+    def rand_(self):
+        with torch.no_grad():
+            self.normal_()
+            self.set_(self.manifold.rand_(self.data))
```

Comment:
I guess this is better to remove self.normal_ from here and put it inside `self.manifold._rand_`
+ just use an inplace version `self.manifold.rand_(self)`